src/services/oauth2.js

```diff
@@ -36,22 +38,24 @@ export class OAuth2Service {
   updateUser(user, data) {
     const idField = this.options.user.idField;
     const id = user[idField];
+    const userService = typeof this._userService === 'string' ? this.app.service(this._userService) : this._userService;
```

Comment:
@marshallswain rather than duplicating this in each method can we DRY it up and put this in the `setup` like I have here: https://github.com/feathersjs/feathers-authentication/blob/0.8/src/services/oauth2.js#L176-L177?